packages/components/src/components/post-header/post-header.tsx

```diff
@@ -305,4 +294,30 @@ export class PostHeader {
       </Host>
     );
   }
+
+  componentDidLoad() {
```

Comment:
I'd rather have the render function at the bottom of the class as we have for other components. Here is the code organization proposed by Stencil: https://stenciljs.com/docs/style-guide#code-organization